### ISSUE TITLE
Handling multiple exotic armor?

### ISSUE THREAD

**Issue by fleeting** [tomcorke/destiny-power-bars#6]
While trying to get raid ready I've been using this to track drops; I had an issue where I had exotic arms equipped (my highest arms) when an exotic chest plate dropped (my highest chest). Power bars of course wouldn't show that new chest plate because I had arms equipped (I assume). This caused my number to be off a little.

I know it shows maximum **equippable** power which is technically right because I can't equip two exotics. Have any thoughts on handling that use case when your goal is power leveling so you only care about the highest in each slot vs the highest you can equip in each slot? Maybe a toggle between those two states?

**Comment by tomcorke** [comment]
Thanks for the feedback!

I've been led to believe that the game also takes this into account when calculating your highest power for drops, so the app looks for the highest combination of gear with the limitation of a single exotic weapon and armour.

I'd be happy to be proven wrong, and will definitely keep an eye out for myself (I think the clearest way would be if I reached a new theoretical average power level with 2 exotics, and a prime engram dropped at that same power) - but given my current understanding of the system I've always been quite quick to infuse that second exotic into a legendary item so it can have an effect for me.

**Comment by fleeting** [comment]
Ahh! That makes complete sense and it does look like that's the way the game calculates it. I just happened to get two exotic armor pieces with decent stats that I didn't want to use either to infuse with.

**Comment by tomcorke** [comment]
Hello again :wave:

I did some more investigation on this, and it seems you're totally right - the game determines your maximum level for the purposes of item drop level (powerfuls etc) with multiple exotics, ignoring the equippable restriction.

I've updated the app to account for this, and when the situation arises it will show the maximum power including both exotics, and you'll see an extra toggle on the character that allows you to view your maximum actual equippable items too.

Thanks again for letting me know!
